ddtrace/mocktracer/mocktracer.go

```diff
@@ -40,13 +41,33 @@ type Tracer interface {
 // to activate the mock tracer. When your test runs, use the returned
 // interface to query the tracer's state.
 func Start() Tracer {
+	return StartWithOptions()
+}
+
+// StartWithOptions is the same as Start, but it allows to specify options.
+func StartWithOptions(opts ...StartOption) Tracer {
```

Comment:
I think only one of `Start` and `StartWithOptions` is needed, eg: func `Start(opts ...StartOption) Tracer`

Reply:
That was my initial idea, then I decided to do a separate func because technically it's a breaking change. If you think it's ok I will add ...opts to Start.